src/current/v23.2/ui-network-latency-page.md

```diff
@@ -33,15 +38,37 @@ For instance, the cluster shown above has nodes in `us-west1`, `us-east1`, and `
 
 ### No connections
 
-Nodes that have lost a connection are displayed in a separate color. This can help you locate a network partition in your cluster.
+Nodes that have completely lost connectivity are color-coded depending on connection status:
+
+- Orange with an exclamation mark indicates `unknown connection state`.
+- Red with a [no symbol](https://www.wikipedia.org/wiki/No_symbol) indicates `Failed connection`. 
+
+This information can help you diagnose a network partition in your cluster.
+
+<img src="{{ 'images/v23.2/ui_network_latency_matrix_suspect_node.png' | relative_url }}" alt="DB Console Network Latency suspect node" style="border:1px solid #eee;max-width:100%" />
+
+Hover over a cell to display more details:
+
+- The direction of the connection marked by `From` and `To`.
+- Locality metadata for origin and destination.
+- Connection status.
+- The error message that resulted from the most recent connection attempt.
+
+This specific information can help you understand the root cause of the connectivity issue.
 
 {{site.data.alerts.callout_info}}
```

Comment:
It seems like this information can be moved out of the admonition and de-duplicated with the info above. I think you have defined a network partition pretty well by this point in the content. Sorry I didn't notice this before.

Reply:
I don't see how to "de-duplicate" the call out. So I am leaving it as is.

Reply:
Apologies for the confusion. I think I was thinking it would be useful to define "network partition" earlier, but now I see that it might be better to use the words early (for SEO), but to move the admonition text into a sub-section. Feel free to ignore these suggestions if you prefer the way it is now. :)